core/storage/base_model/gae_models.py

```diff
@@ -84,47 +84,32 @@
 
 
 class BaseModel(datastore_services.Model):
-    """Base model for all persistent object storage classes."""
+    """Base model for all persistent object storage classes.
+
+    IMPORTANT: If humans can interface with the model directly, then inherit
+    from BaseHumanMaintainedModel instead and use the put_for_human family of
+    methods when appropriate.
+    """
 
     # Specifies whether the model's id is used as a key in Takeout. By default,
     # the model's id is not used as the key for the Takeout dict.
     ID_IS_USED_AS_TAKEOUT_KEY = False
 
-    # When this entity was first created. This value should only be modified by
-    # the update_timestamps method.
+    # When this entity was first created. This value should not be modified.
     created_on = (
-        datastore_services.DateTimeProperty(indexed=True, required=True))
-    # When this entity was last updated. This value should only be modified by
-    # the update_timestamps method.
+        datastore_services.DateTimeProperty(required=True, indexed=True))
+    # When this entity was last updated. This value should not be modified.
```

Comment:
Suggest clarifying that it should only be modified by the class implementation itself & nowhere else. The current documentation is a bit unexpected since the field is actually being updated.

Reply:
Done.